components/rating/rating.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
you can also add one more meta tag here, check the image
![image](https://user-images.githubusercontent.com/72219676/152690035-f2010036-c7a9-4aba-b376-f9dc84d56732.png)

Reply:
Ohh I didn't know this, will make this change.